### PR TITLE
Fixes #13813 - Removed "\ " so ruby 1.8 could read the ENC

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: app/controllers/hosts_controller.rb [hunk 1]
```diff
@@ -197,7 +197,7 @@ def externalNodes
     begin
       respond_to do |format|
         format.html { render :text => "<pre>#{ERB::Util.html_escape(@host.info.to_yaml)}</pre>" }
-        format.yml { render :text => @host.info.to_yaml }
+        format.yml { render :text => @host.info.to_yaml(:line_width => -1) }
```
Comment (iNecas) [inline app/controllers/hosts_controller.rb]: we should do this both fore html and yml version so have identical results

**Comment by ShimShtein** [comment]
Fixed commit message and added fix for html output.

File: app/controllers/hosts_controller.rb [hunk 2]
```diff
@@ -196,8 +196,8 @@ def externalNodes
 
     begin
       respond_to do |format|
-        format.html { render :text => "<pre>#{ERB::Util.html_escape(@host.info.to_yaml)}</pre>" }
-        format.yml { render :text => @host.info.to_yaml }
+        format.html { render :text => "<pre>#{ERB::Util.html_escape(@host.info.to_yaml(:line_width => -1))}</pre>" }
+        format.yml { render :text => @host.info.to_yaml(:line_width => -1) }
```
Comment (iNecas) [inline app/controllers/hosts_controller.rb]: i would suggest extracting the yaml into variable to reduce code duplicities. Good to merge after that.

**Comment by ShimShtein** [comment]
Removed duplication.

**Comment by iNecas** [comment]
Tested and it works well, Thanks @ShimShtein, merged as 0069c98dcb50a8960f92397921cfb8a98dc8401e

**Pull request closed without merging**